### PR TITLE
Fix CA1307 warning

### PR DESCRIPTION
**Pull request by thomaslevesque**
Introduced when `netcoreapp2.1` targets were added to tests.

### CODE REVIEW

File: tests/FakeItEasy.IntegrationTests/StringArgumentConstraints.cs [hunk 1]
```diff
@@ -11,7 +11,11 @@ public static string StartsWith(this IArgumentConstraintManager<string> scope, s
 
         public static string Contains(this IArgumentConstraintManager<string> scope, string value)
         {
+#if FEATURE_STRING_CONTAINS_COMPARISONTYPE
+            return scope.Matches(x => x.Contains(value, StringComparison.CurrentCulture), $@"Contains ""{value}""");
```
Comment (thomaslevesque) [inline tests/FakeItEasy.IntegrationTests/StringArgumentConstraints.cs]: `String.Contains` now has multiple overloads. Not specifying the comparison type means current culture is used implicitly, which could be an issue and triggers CA1307. I specified current culture explicitly to keep the same behavior.

**Comment by thomaslevesque** [comment]
@blairconrad the warning was in tests, so I could just suppress it if you prefer.

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque. I noticed this on Friday and hadn't yet gotten back to it.

**Comment by thomaslevesque** [comment]
Wow, you're fast @blairconrad!

**Comment by thomaslevesque** [comment]
Thanks @blairconrad!

**Comment by blairconrad** [comment]
> Wow, you're fast @blairconrad!

Luck. I just happened to see the e-mail for the PR come in.

**Comment by blairconrad** [comment]
This change has been released as part of [FakeItEasy 4.9.0](https://github.com/FakeItEasy/FakeItEasy/releases/4.9.0).

**Pull request merged**